Softeq.XToolkit.WhiteLabel/Mvvm/ViewModelBase.cs

```diff
@@ -5,39 +5,49 @@
 
 namespace Softeq.XToolkit.WhiteLabel.Mvvm
 {
-    public interface IViewModelBase
-    {
-        bool IsInitialized { get; }
-
-        void OnInitialize();
-        void OnAppearing();
-        void OnDisappearing();
-    }
-
     /// <summary>
     ///     A base class for the ViewModel classes in the MVVM pattern.
+    ///     Each ViewModel should correspond to the page or some other UI component.
     /// </summary>
     public abstract class ViewModelBase : ObservableObject, IViewModelBase
     {
         private bool _isBusy;
 
+        /// <summary>
+        ///     Gets or sets a value indicating whether this ViewModel is doing some work.
+        /// </summary>
         public bool IsBusy
         {
             get => _isBusy;
             set => Set(ref _isBusy, value);
         }
 
+        /// <summary>
+        ///     Gets a value indicating whether this ViewModel is initialized.
+        /// </summary>
         public bool IsInitialized { get; private set; }
 
+        /// <summary>
+        ///     Called when the page or UI component corresponding to this ViewModel
+        ///     is initializing.
+        /// </summary>
```

Comment:
Could you please move docs to the interface and use `<inheritdoc>` here?

Reply:
Sure thing, done!